browser/js/admin/admin.edit/admin.create.state.js

```diff
@@ -0,0 +1,15 @@
+app.config(function ($stateProvider) {
+    $stateProvider.state('adminCreateProduct', {
+        url: '/admin/create',
+        controller: 'AdminCreateCtrl',
+        templateUrl: '/js/admin/admin.edit/admin.create.template.html',
+        resolve: {
+            isAdminUser: function(AuthService, $state){
+                AuthService.getLoggedInUser()
```

Comment:
Consider doing this logic somewhere in the user factory, so that it only needs to be written out once and we won't have to pass AuthService into every resolve block